### PR TITLE
UIORGS-76 Permissions problems with ui-orgs

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Permissions problems with ui-organizations
https://issues.folio.org/browse/UIORGS-76
## Approach
Fix permission name for `Organizations: all permissions for Create, View, Edit and Delete organizations`
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -138,12 +138,12 @@
...
       },
       {
-        "permissionName": "ui-organizations.all",
+        "permissionName": "ui-organizations.module.all",
         "displayName": "Organizations: all permissions for Create, View, Edit and Delete organizations",
         "visible": true,
         "description": "",
         "subPermissions": [
-          "organizations-storage.all"
+          "ui-organizations.module.all"
```
Comment (aliaksei-chumakou) [inline package.json]: isn't it a circular dependency? You have sub perm `ui-organizations.module.all` inside perm `ui-organizations.module.all`

Comment (aliaksei-chumakou) [reply package.json]: maybe you meant https://github.com/folio-org/mod-organizations-storage/blob/master/descriptors/ModuleDescriptor-template.json#L574

File: package.json [hunk 2]
```diff
@@ -138,12 +138,12 @@
         ]
       },
       {
-        "permissionName": "ui-organizations.all",
+        "permissionName": "ui-organizations.module.all",
```
Comment (aliaksei-chumakou) [inline package.json]: I don't think you should rename this

File: package.json (hunk 1 above)
Comment (AndreiShumski2) [reply package.json]: My fault. Fixed

**Pull request merged**